homeassistant/components/switchbot_cloud/sensor.py

```diff
@@ -89,6 +93,20 @@
     native_unit_of_measurement=CONCENTRATION_PARTS_PER_MILLION,
 )
 
+POWER_CONSUMPTION_DESCRIPTION = SensorEntityDescription(
+    key=SENSOR_TYPE_POWER_CONSUMPTION,
+    device_class=SensorDeviceClass.ENERGY,
+    state_class=SensorStateClass.TOTAL_INCREASING,
+    native_unit_of_measurement=UnitOfEnergy.MILLIWATT_HOUR,
+)
+
+DURATION_OF_USED_DESCRIPTION = SensorEntityDescription(
+    key=SENSOR_TYPE_DURATION_OF_USED,
+    device_class=SensorDeviceClass.DURATION,
+    state_class=SensorStateClass.MEASUREMENT,
+    native_unit_of_measurement=UnitOfTime.MINUTES,
+)
```

Comment:
What does this mean? What does this expose?

Reply:
the duration that the device has been used during a day, measured in minutes

Reply:
1. Should we disable this by default as it writes state quite a lot?
2. Should we set the state class to TOTAL_INCREASING? As it will only go up or go to 0?